pkg/virt-operator/resource/generate/components/crds_test.go

```diff
@@ -119,4 +137,316 @@ var _ = Describe("CRDs", func() {
 		Entry("for VirtualMachineClone", NewVirtualMachineCloneCrd, "Phase", "SourceVirtualMachine", "TargetVirtualMachine"),
 		Entry("for MigrationPolicy", NewMigrationPolicyCrd),
 	)
+
+	Context("Additional printer columns map to expected value", func() {
+		It("for VirtualMachineInstance", func() {
+			vmi := &v1.VirtualMachineInstance{
+				ObjectMeta: metav1.ObjectMeta{
+					CreationTimestamp: metav1.Time{Time: time.Now()},
+				},
+				Status: v1.VirtualMachineInstanceStatus{
+					Phase: v1.Running,
+					Interfaces: []v1.VirtualMachineInstanceNetworkInterface{
+						{
+							IP: "1.2.3.4",
+						},
+					},
+					NodeName: "test-node",
+					Conditions: []v1.VirtualMachineInstanceCondition{
+						{
+							Type:   "Ready",
+							Status: "True",
+						},
+						{
+							Type:   "LiveMigratable",
+							Status: "False",
+						},
+						{
+							Type:   "Paused",
+							Status: "False",
+						},
+					},
+				},
+			}
+
+			crd, err := NewVirtualMachineInstanceCrd()
+			Expect(err).ToNot(HaveOccurred())
+
+			obj := serialize(vmi)
+			for _, version := range crd.Spec.Versions {
+				Expect(extract(version.AdditionalPrinterColumns[0].JSONPath, obj)).To(Equal(vmi.ObjectMeta.CreationTimestamp.UTC().Format(time.RFC3339)))
+				Expect(extract(version.AdditionalPrinterColumns[1].JSONPath, obj)).To(Equal(string(vmi.Status.Phase)))
+				Expect(extract(version.AdditionalPrinterColumns[2].JSONPath, obj)).To(Equal(vmi.Status.Interfaces[0].IP))
+				Expect(extract(version.AdditionalPrinterColumns[3].JSONPath, obj)).To(Equal(vmi.Status.NodeName))
+				Expect(extract(version.AdditionalPrinterColumns[4].JSONPath, obj)).To(Equal(string(vmi.Status.Conditions[0].Status)))
+				Expect(extract(version.AdditionalPrinterColumns[5].JSONPath, obj)).To(Equal(string(vmi.Status.Conditions[1].Status)))
+				Expect(extract(version.AdditionalPrinterColumns[6].JSONPath, obj)).To(Equal(string(vmi.Status.Conditions[2].Status)))
```

Comment:
nit: I would assert to concrete values and probably iterate over the `AdditionalPrinterColumns` to make it detect deletion or addition